### PR TITLE
latexpdf

### PR DESCRIPTION
**Pull request by renich**
It was a pain figuring out exactly which tex dependencies this target required. 

This is what is required.

### CODE REVIEW

**Comment by jackorp** [comment]
Thanks for this! A thought, there are meta packages regarding latex that help you take in deps:
```
$ dnf search "texlive-scheme*"
Last metadata expiration check: 1 day, 19:53:33 ago on Mon 27 Feb 2023 02:42:46 PM CET.
========================================================================================== Name Matched: texlive-scheme* ===========================================================================================
texlive-scheme-basic.noarch : basic scheme (plain and latex)
texlive-scheme-context.noarch : ConTeXt scheme
texlive-scheme-full.noarch : full scheme (everything)
texlive-scheme-gust.noarch : GUST TeX Live scheme
texlive-scheme-medium.noarch : medium scheme (small + more packages and languages)
texlive-scheme-minimal.noarch : minimal scheme (plain only)
texlive-scheme-small.noarch : small scheme (basic + xetex, metapost, a few languages)
texlive-scheme-tetex.noarch : teTeX scheme (more than medium, but nowhere near full)
```

Using a texlive-scheme-* is IMO more comfortable, but it might bring in more than you might ever need. This might be especially true with sphinx.

Second thing: you don't probably need to specify "python3-sphinx" for that latexmk target. We can assume that package is installed already, as that is the first step in the article.

**Comment by jackorp** [comment]
I also noticed "texlive-babel-spanish", this is generally some language specific typesetting. My question would be, should I use the babel-spanish and then go and use for example Czech language, will it still work or will I need "texlive-babel-czech"? :) that is where the package collections of texlive-scheme come useful IMO.

**Comment by renich** [comment]
You're right @jackorp, my bad. I'll remove that. I was generating some documentation in Spanish when I was figuring this out.

**Comment by renich** [comment]
Done.

**Comment by jackorp** [comment]
Thanks for fixing that, however WDYT WRT removing python3-sphinx and the texlive-scheme-* package?

**Comment by renich** [comment]
> Thanks for fixing that, however WDYT WRT removing python3-sphinx and the texlive-scheme-* package?

Let me just test the outcome and if it works, I'll gladly remove it.

**Comment by renich** [comment]
OK, I've tested in a fedora 37 container and it seems to work. I had to add a part where I explain how to add language support for your project's language. 

I couldn't find texlive-scheme-* though.

**Comment by jackorp** [comment]
> I couldn't find texlive-scheme-* though.

Hmm that's weird I get above 500 packages to install if I enter `sudo dnf install texlive-scheme-basic` in a Fedora 37 container (it is a lot but LaTeX is not THAT simple to setup). JFTR the texlive-scheme-full installs around 2.5G of packages, there are around 4k packages to install that way :D

File: tech/languages/python/sphinx.md [hunk 1]
```diff
@@ -68,6 +68,25 @@ Please use `make <target>' where <target> is one of
...
 
+## latexpdf
+
+In order to be able to run the `latexpdf` target, you will need to install the following dependencies:
+
+``` sh
+dnf -y install latexmk texlive-capt-of texlive-cmap texlive-ec texlive-fncychap texlive-framed texlive-makeindex texlive-metafont texlive-needspace texlive-parskip texlive-tabulary texlive-tex-gyre texlive-upquote texlive-wrapfig
+```
+
+Additionally, you need to install babel support for your project's language. for example, if you're default language is English, install `texlive-babel-english`:
```
Comment (jackorp) [inline tech/languages/python/sphinx.md:79]:
```suggestion
Additionally, you need to install babel support for your project's language. for example, if your default language is English, install `texlive-babel-english`:
```

**Review by jackorp — CHANGES_REQUESTED** [review]
Minor grammar nit :)

**Comment by jackorp** [comment]
I like the babel section. Though for some reason, other languages that make use of different encoding might require other packages (I remember teammate from another project struggling with enabling czech for one LaTeX project).

But for now, I'd say it is enough to get it working, let's not overthink it. I'll take you by your word that it looks OK with the package set you have found.

LGTM from my side, if you're ok with it I think we can merge now.

**Comment by renich** [comment]
For what is worth, I tried using `texlive-scheme-basic` and `texlive-scheme-minimal` and, still, I had to install:

* texlive-capt-of
* texlive-cmap
* texlive-fncychap                  
* texlive-framed                   
* texlive-needspace               
* texlive-parskip                
* texlive-tabulary              
* texlive-tex-gyre             
* texlive-upquote             
* texlive-wrapfig

**Comment by renich** [comment]
Thank you @jackorp. :)

**Pull request merged**